redirects/17.0.txt

```diff
@@ -11,6 +11,23 @@ applications/inventory_and_mrp/inventory/warehouses_storage/advanced_operations_
+applications/inventory_and_mrp/manufacturing/management/subcontracting.rst applications/inventory_and_mrp/manufacturing/workflows/subcontracting.rst                                        # management/* --> workflows/*
+applications/inventory_and_mrp/manufacturing/management/use_mps.rst applications/inventory_and_mrp/manufacturing/workflows/use_mps.rst                                                      # management/* --> workflows/*
+applications/inventory_and_mrp/manufacturing/management/using_work_centers.rst applications/inventory_and_mrp/manufacturing/advanced_configuration/using_work_centers.rst                   # management/* --> advanced_configuration/*
+applications/inventory_and_mrp/manufacturing/management/work_center_time_off.rst applications/inventory_and_mrp/manufacturing/workflows/work_center_time_off.rst                            # management/* --> workflows/*
+applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst applications/inventory_and_mrp/manufacturing/workflows/scrap_manufacturing.rst                              # management/* --> workflows/*
+applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst applications/inventory_and_mrp/manufacturing/workflows/manufacturing_backorders.rst                    # management/* --> workflows/*
+applications/inventory_and_mrp/manufacturing/management/configure_manufacturing_product.rst applications/inventory_and_mrp/manufacturing/basic_setup/configure_manufacturing_product.rst    # management/* --> basic_setup/*
+applications/inventory_and_mrp/manufacturing/management/split_merge.rst applications/inventory_and_mrp/manufacturing/workflows/split_merge.rst                                              # management/* --> workflows/*
+applications/inventory_and_mrp/manufacturing/management/work_order_dependencies.rst applications/inventory_and_mrp/manufacturing/advanced_configuration/work_order_dependencies.rst         # management/* --> advanced_configuration/*
+applications/inventory_and_mrp/manufacturing/management/unbuild_orders.rst applications/inventory_and_mrp/manufacturing/workflows/unbuild_orders.rst                                        # management/* --> workflows/*
+applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst applications/inventory_and_mrp/manufacturing/basic_setup/one_step_manufacturing.rst                      # management/* --> basic_setup/*
+applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst applications/inventory_and_mrp/manufacturing/basic_setup/two_step_manufacturing.rst                      # management/* --> basic_setup/*
+applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst applications/inventory_and_mrp/manufacturing/basic_setup/three_step_manufacturing.rst                  # management/* --> basic_setup/*
```

Comment:
Update the order to be [alphabetical](https://github.com/odoo/documentation/blob/17.0/redirects/MANUAL.md#how-do-i-create-a-redirect-rule) :)
```suggestion
applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst applications/inventory_and_mrp/manufacturing/basic_setup/bill_configuration.rst                              # management/* --> basic_setup/*
applications/inventory_and_mrp/manufacturing/management/configure_manufacturing_product.rst applications/inventory_and_mrp/manufacturing/basic_setup/configure_manufacturing_product.rst    # management/* --> basic_setup/*
applications/inventory_and_mrp/manufacturing/management/kit_shipping.rst applications/inventory_and_mrp/manufacturing/advanced_configuration/kit_shipping.rst                               # management/* --> advanced_configuration/*
applications/inventory_and_mrp/manufacturing/management/manufacturing_backorders.rst applications/inventory_and_mrp/manufacturing/workflows/manufacturing_backorders.rst                    # management/* --> workflows/*
applications/inventory_and_mrp/manufacturing/management/one_step_manufacturing.rst applications/inventory_and_mrp/manufacturing/basic_setup/one_step_manufacturing.rst                      # management/* --> basic_setup/*
applications/inventory_and_mrp/manufacturing/management/product_variants.rst applications/inventory_and_mrp/manufacturing/advanced_configuration/product_variants.rst                       # management/* --> advanced_configuration/*
applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst applications/inventory_and_mrp/manufacturing/workflows/scrap_manufacturing.rst                              # management/* --> workflows/*
applications/inventory_and_mrp/manufacturing/management/split_merge.rst applications/inventory_and_mrp/manufacturing/workflows/split_merge.rst                                              # management/* --> workflows/*
applications/inventory_and_mrp/manufacturing/management/sub_assemblies.rst applications/inventory_and_mrp/manufacturing/advanced_configuration/sub_assemblies.rst                           # management/* --> advanced_configuration/*
applications/inventory_and_mrp/manufacturing/management/subcontracting.rst applications/inventory_and_mrp/manufacturing/workflows/subcontracting.rst                                        # management/* --> workflows/*
applications/inventory_and_mrp/manufacturing/management/two_step_manufacturing.rst applications/inventory_and_mrp/manufacturing/basic_setup/two_step_manufacturing.rst                      # management/* --> basic_setup/*
applications/inventory_and_mrp/manufacturing/management/three_step_manufacturing.rst applications/inventory_and_mrp/manufacturing/basic_setup/three_step_manufacturing.rst                  # management/* --> basic_setup/*
applications/inventory_and_mrp/manufacturing/management/unbuild_orders.rst applications/inventory_and_mrp/manufacturing/workflows/unbuild_orders.rst                                        # management/* --> workflows/*
applications/inventory_and_mrp/manufacturing/management/use_mps.rst applications/inventory_and_mrp/manufacturing/workflows/use_mps.rst                                                      # management/* --> workflows/*
applications/inventory_and_mrp/manufacturing/management/using_work_centers.rst applications/inventory_and_mrp/manufacturing/advanced_configuration/using_work_centers.rst                   # management/* --> advanced_configuration/*
applications/inventory_and_mrp/manufacturing/management/work_center_time_off.rst applications/inventory_and_mrp/manufacturing/workflows/work_center_time_off.rst                            # management/* --> workflows/*
applications/inventory_and_mrp/manufacturing/management/work_order_dependencies.rst applications/inventory_and_mrp/manufacturing/advanced_configuration/work_order_dependencies.rst         # management/* --> advanced_configuration/*
```